.github/workflows/fuzz.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
         # commit hash == v1.2.0
-      - uses: jidicula/go-fuzz-action@4f24eed45b25214f31a9fe035ca68ea2c88c6a13
+      - uses: jidicula/go-fuzz-action@4f24eed45b25214f31a9fe035ca68ea2c88c6a13 # TSCCR: no entry for action: "jidicula/go-fuzz-action"
```

Comment:
@jimlambrt I don't know how much is different here, but not sure if we want to modify this workflow to use the modifications that were made in `boundary`: https://github.com/hashicorp/boundary/blob/main/.github/workflows/fuzz.yml

Reply:
I saw Johan's work, but I've had no problems downloading the data set on failures.

Reply:
I think it also swapped some actions so it's not using any "untrusted" actions?

Reply:
I opened a PR to get it trusted.